### PR TITLE
Tweak label prop of Checkbox component to accept node

### PR DESCRIPTION
**Pull request by dozortsev**

### CODE REVIEW

File: src/checkbox/Checkbox.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ Checkbox.propTypes = {
   checked: Types.bool,
   required: Types.bool,
   disabled: Types.bool,
-  label: Types.string.isRequired,
+  label: Types.node.isRequired,
```
Comment (andreapiras) [inline src/checkbox/Checkbox.js]: this is a breaking change. Can it not accept both?

**Comment by andreapiras** [comment]
Tests are failing :)

File: src/checkbox/Checkbox.js (hunk 1 above)
Comment (dozortsev) [reply src/checkbox/Checkbox.js]: `node` already does this, check the [doc](https://reactjs.org/docs/typechecking-with-proptypes.html) 🙂

> // Anything that can be rendered: numbers, strings, elements or an array

**Comment by dozortsev** [comment]
just a little code formatting issue

**Pull request merged**